### PR TITLE
Hessians for optimization data set

### PR DESCRIPTION
**Pull request by jmaat**
@dgasmith These are hessians from the recently submitted gen 2 optimization data sets.

### CODE REVIEW

File: 2020-03-20-OpenFF-Gen-2-Hessian-Set-1-Roche/03_create_hessian_dataset.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+import copy
```
Comment (dgasmith) [inline 2020-03-20-OpenFF-Gen-2-Hessian-Set-1-Roche/03_create_hessian_dataset.py]: We should add these to the pre-existing folders I think as they are still steps in the same dataset. If we continue with the separate dataset strategy we should not duplicate the first steps.

Comment (jmaat) [reply 2020-03-20-OpenFF-Gen-2-Hessian-Set-1-Roche/03_create_hessian_dataset.py]: Ok, removed the additional directories and added `03_create_hessian_dataset.py` to preexisting optimization directories. @dgasmith

**Comment by dgasmith** [comment]
It looks like the Hessian directors are still in the PR.

**Comment by jmaat** [comment]
Hessian directories should be removed now, @dgasmith

**Pull request merged**